### ISSUE TITLE
Quality Comments

### ISSUE THREAD

**Issue by mmills2** [sloanelauren/QUBetterParking#4]
The comments in the repo are very good, and I think we should continue to keep commenting like this.

**Comment by sloanelauren** [comment]
I agree it's been so helpful to quickly understand what another collaborator has been writing!

**Comment by AnzilottiChristopher** [comment]
The creators of this repo did a great job explaining what their code does.
